utils/zpe-updater/cmd/tools/main.go

```diff
@@ -43,6 +52,12 @@ func main() {
 	}
 
 	if logFile != "" {
+		err := ioutil.WriteFile(logFile, []byte("Starting zpe-updater"), 0755)
```

Comment:
This will truncate the file every time is called so that's not want we want. You want to make sure you open the file in append mode and not truncate it